layouts/project/single.html

```diff
@@ -20,19 +20,19 @@
         <img class="about__image" src="{{ .Params.article_image}}" alt="hal" />
       </div>
       <div class="about__block">
-        <div class="about__descr-title">Kilka słów więcej</div>
         <div class="about__descr">{{.Params.short_description | safeHTML }}
+          {{.Content}}
         </div>
       </div>
     </div>
 
-    <div class="about__wrapper">
+    <!-- <div class="about__wrapper">
       <div class="about__block">
         <div class="about__descr">
           {{.Content}}
         </div>
       </div>
-    </div>
+    </div> -->
```

Comment:
Nie wiem, czemu zmienione zostało położenie contentu.
Teraz to wygląda tak: 
![image](https://user-images.githubusercontent.com/45544416/226970421-5d8469ef-7e41-45f3-b357-6f3d274c137a.png)